### PR TITLE
spes: Initial crDroid 9.0 build for Redmi Note 11|11 NFC

### PR DESCRIPTION
**Pull request by Jabiyeff**
Signed-off-by: Jabiyeff <cebiyevanar@gmail.com>

### CODE REVIEW

File: spes.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        "device": "Redmi Note 11",
+        "filename": "crDroidAndroid-13.0-20221106-spes-v9.0.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/spes/9.x/crDroidAndroid-13.0-20221106-spes-v9.0.zip/download",
+        "timestamp": 1667722116,
+        "md5": "d5afbd87714c27fd594beb52f1ecf5b5",
+        "sha256": "a2ed203c4a40b96c315049f6943028f585469599951d43f41ca0fc554fb37e89",
+        "size": 1199972504,
+        "version": "9.0",
+        "buildtype": "Weekly",
+        "forum": "https://forum.xda-developers.com/t/rom-official-12-1-crdroidandroid-8-x-redmi-note-11-spes-spesn.4440747",
```
Comment (gwolf2u) [inline spes.json:15]: Update forum info

File: spes.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        "gapps": "",
+        "firmware": "https://xiaomifirmwareupdater.com/firmware/spes/",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "",
+        "paypal": "https://www.patreon.com/jabiyeff",
+        "telegram": "https://t.me/crDroidSpes",
+        "dt": "https://github.com/crdroidandroid/android_device_xiaomi_spes",
+        "common-dt": "",
+        "kernel": ""
```
Comment (gwolf2u) [inline spes.json:25]: Kernel?

File: spes.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "forum": "https://forum.xda-developers.com/t/rom-official-12-1-crdroidandroid-8-x-redmi-note-11-spes-spesn.4440747",
+        "gapps": "",
+        "firmware": "https://xiaomifirmwareupdater.com/firmware/spes/",
+        "modem": "",
+        "bootloader": "",
+        "recovery": "",
+        "paypal": "https://www.patreon.com/jabiyeff",
+        "telegram": "https://t.me/crDroidSpes",
+        "dt": "https://github.com/crdroidandroid/android_device_xiaomi_spes",
+        "common-dt": "",
```
Comment (gwolf2u) [inline spes.json:24]: No 13.0 branch

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: spes.json (hunk 2 above)
Comment (Jabiyeff) [reply spes.json:25]: Its prebuilt

File: spes.json (hunk 3 above)
Comment (Jabiyeff) [reply spes.json:24]: Will push now

File: spes.json (hunk 1 above)
Comment (Jabiyeff) [reply spes.json:15]: Will do

Comment (Jabiyeff) [reply spes.json:15]: > Update forum info

Done

File: spes.json (hunk 3 above)
Comment (Jabiyeff) [reply spes.json:24]: > No 13.0 branch

Pushed

File: spes.json (hunk 1 above)
Comment (gwolf2u) [reply spes.json:15]: set changelog and download page url to website please
dl: https://crdroid.net/spes/9
changelog: https://crdroid.net/spes/9#changelog

Comment (Jabiyeff) [reply spes.json:15]: Sure, I already using download link from crDroid website on telegram releases. Also fixed now on XDA forum.

**Pull request merged**